### DISCUSSION TITLE
How can I have introspect include schemas besides public?

### DISCUSSION THREAD

**Discussion by giraffesyo** [prisma/prisma#3135] [Q&A] [answered]
Hello, I've been using Prisma and I really love it! I have a question about schemas.

I have the following postgres database:

```sql
CREATE SCHEMA bot;

CREATE TABLE bot.WatchedChannel (
  id text not null,
  PRIMARY KEY (id)
);
CREATE TABLE bot.FeedChannel (
  id text not null,
  PRIMARY KEY (id)
);
CREATE TABLE bot.Administrator (
  id text not null,
  PRIMARY KEY (id)
);
```

When I introspect, it only makes models for the `public` schema, which is the default schema in postgres. 

Is there a way to tell prisma to introspect all schemas (in this case the `bot` schema)?

**Comment by janpio** [comment, marked as answer]
Just add `?schema=bot` to your Postgres connection string.

https://www.prisma.io/docs/reference/database-connectors/postgresql#connection-url

**Reply by giraffesyo** [reply]
Thanks, that worked!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
